shapely/_geometry.py

```diff
@@ -54,8 +54,9 @@ class GeometryType(IntEnum):
 
 @multithreading_enabled
 def get_type_id(geometry, **kwargs):
-    """Returns the type ID of a geometry.
+    """Return the type ID of a geometry.
 
+    Possible values are:
     - None (missing) is -1
```

Comment:
It's unfortunate that pydocstyle doesn't catch that, but there needs to be a blank line between those two lines (a blank line before and after the bullet point list is used for separation of the different blocks of formatting)

(see the current misformatting in https://shapely.readthedocs.io/en/latest/reference/shapely.get_type_id.html as a result)